main.js

```diff
@@ -7,21 +7,22 @@ let student;
 student = 'Daria Ramanowska';
 console.log(student);
 
-user = student; //переопределены данные на 'Daria Ramanowska'
+user = student; //redefined data to 'Daria Ramanowska'
 console.log(user);
 
 let test = 1;
 test++;
 
-console.log(test + '1'); // '21' string
+console.log(test + "1"); // '21' string
```

Comment:
It is better to use a consistent style for the strings. I think prettier may solve it.